.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think this a local file has been committed by accident, I don't think we want this to be tracked by git. You can untrack the file with
`git rm --cached .project`

This will leave the local file on your machine untouched but tells git you don't want it to be included in the remote repository once you have committed this change.